Octokit/Helpers/ApiUrls.cs

```diff
@@ -1834,6 +1834,22 @@ public static Uri RepositoryTags(string owner, string name)
             return "repos/{0}/{1}/tags".FormatUri(owner, name);
         }
 
+        /// <summary>
+        /// Returns the <see cref="Uri"/> for a repository transfer.
+        /// </summary>
+        /// <param name="currentOwner">The current owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <returns></returns>
+        public static Uri RepositoryTransfer(string currentOwner, string name)
+        {
+            return "repos/{0}/{1}/transfer".FormatUri(currentOwner, name);
+        }
+
+        public static Uri RepositoryTransfer(long repositoryId)
```

Comment:
If the above owner/name method has XmlDoc comments provided then we should add them for this repositoryId one too